debian/control

```diff
@@ -61,6 +61,7 @@ Description: messaging application for Ubuntu
  An application to send and receive messages.
 
 Package: messaging-app-autopilot
+Build-Profiles: <!noinsttest>
```

Comment:
We already have `nocheck` build profile, don't we? Crossbuilder already set this profile.

Reply:
We do. But I don't think `nocheck` should avoid building and installing tests in the debian packaging (and neither should cross-compiling prevent it either, I think). Rather, we should simply not run them in that case.

Reply:
As @dobey wrote, it's two different things. You can read the description of the `noinsttest` profile [here](https://wiki.debian.org/BuildProfileSpec), and I believe it's appropriate for autopilot tests.

Reply:
Ahh, I missed the fact that `noinsttest` build profile is a thing. Then that's Ok.